Update markdown documents (#751)
terraform-ibm-modules/terraform-ibm-powervs-infrastructure: solutions/extension/README.md

# IBM Cloud Catalog Solution for Power Virtual Server with VPC Landing Zone Extension Variation

This example extends an existing PowerVS infrastructure for deployable architectures deployed as full-stack with an additional PowerVS workspace.
It provisions the following infrastructure on top of the deployed Full Stack solution:

- A **Power Virtual Server workspace** with the following network topology:
    - Creates two private networks: a management network and a backup network
    - Creates one or two IBM Cloud connections in a Non-PER environment.
    - Attaches the private networks to the IBM Cloud connections in a Non-PER environment.
    - Attaches the IBM Cloud connections to a transit gateway in a Non-PER environment.
    - Attaches the PowerVS workspace to Transit gateway in PER-enabled DC
    - Creates an SSH key.

### Notes:

> - Please ensure that you are selecting a PowerVS zone different from the one used for the prerequisite infrastructure.

Reviewer: ```suggestion
- Make sure that you select a PowerVS zone that is different from the one used for the prerequisite infrastructure.
```
